database/database_test.go

```diff
@@ -35,6 +35,7 @@ func TestDatabase(t *testing.T) {
 		"testAddressIndex":      testAddressIndex,
 		"testDeleteTicket":      testDeleteTicket,
 		"testVoteChangeRecords": testVoteChangeRecords,
+		"testCountTickets":		 testCountTickets,
```

Comment:
- Not go fmt'ed.
- I see changes to other files that don't seem related to this PR.